pkg/storage/azure/azure.go

```diff
@@ -360,13 +354,26 @@ func (d *driver) StorageExists(cr *imageregistryv1.Config) (bool, error) {
 	_, err = container.GetProperties(d.Context, azblob.LeaseAccessConditions{})
 	if e, ok := err.(azblob.StorageError); ok {
 		if e.ServiceCode() == azblob.ServiceCodeContainerNotFound {
-			util.UpdateCondition(cr, imageregistryv1.StorageExists, operatorapiv1.ConditionFalse, storageExistsReasonContainerNotFound, "Container does not exist")
-			return false, nil
+			return nil
```

Comment:
The return value is ambiguous now. Does it mean "exists" or "doesn't exist"?

Reply:
updated in next push